### ISSUE TITLE
Prediction variable name in `Calibrated_spike_inference_with_Cascade.ipynb`

### ISSUE THREAD

**Issue by kimtonyhyun** [HelmchenLabSoftware/Cascade#20]
First of all, thanks for sharing this exciting tool for Ca2+ data analysis.

I have a minor question/comment regarding a variable name in the demo script `Calibrated_spike_inference_with_Cascade.ipynb`.

In Step 8, the result of `cascade.predict(...)` is stored in a variable called `spike_rates`. In Step 11, this variable is stored into a MAT file with the variable name `spike_rates` as well.

However, the first entry of the FAQ indicates that the output of the algorithm is the _probability_ of spikes at the same resolution as the Ca2+ recording. It states that we can convert the output into the instantaneous spike rate by multiplying with the frame rate.

So, I wanted to confirm that the variable `spike_rate` that is returned by `cascade.predict(...)` is indeed spike probabilities, and not rates (i.e. the `predict` function does not multiply by the frame rate internally). I think this is the case, but I wanted to double check. If so, it might be clearer for users to label the output of `cascade.predict` to be `spike_prob` (or something similar).

**Comment by PTRRupprecht** [comment]
Hi Tony,

You're absolutely right, the `spike_rates` variable name is misleading. I had not noticed before. I had chosen this variable name because we used "spike rates" but rarely "spike probabilities" in the paper.

However, `spike_prob` is indeed a better name choice. I have now applied your suggested change in the relevant places of the code and the FAQ, and I hope it will not cause any backward compatibility issues for anybody.

Best wishes,
Peter
